core/src/test/resources/schema/postgres/postgres-schema-v5.sql

```diff
@@ -0,0 +1,53 @@
+DROP TABLE IF EXISTS public.event_tag;
+DROP TABLE IF EXISTS public.journal_event;
+
+CREATE TABLE IF NOT EXISTS public.journal_event (
```

Comment:
alternative table name could be `event_journal` to pair with `event_tag`, but I don't have a strong opinion

Reply:
👍